src/main/telemetry/crsf.c

```diff
@@ -195,24 +195,24 @@ void crsfFrameBatterySensor(sbuf_t *dst)
 }
 
 typedef enum {
-    CRSF_ACTIVE_ANTENNA1 = 0,
-    CRSF_ACTIVE_ANTENNA2 = 1,
+    CRSF_ACTIVE_ANTENNA1,
+    CRSF_ACTIVE_ANTENNA2,
 } crsfActiveAntenna_e;
 
 typedef enum {
-    CRSF_RF_MODE_4_HZ = 0,
-    CRSF_RF_MODE_50_HZ = 1,
-    CRSF_RF_MODE_150_HZ = 2,
+    CRSF_RF_MODE_4_HZ,
+    CRSF_RF_MODE_50_HZ,
+    CRSF_RF_MODE_150_HZ,
 } crsrRfMode_e;
 
 typedef enum {
-    CRSF_RF_POWER_0_mW = 0,
-    CRSF_RF_POWER_10_mW = 1,
-    CRSF_RF_POWER_25_mW = 2,
-    CRSF_RF_POWER_100_mW = 3,
-    CRSF_RF_POWER_500_mW = 4,
-    CRSF_RF_POWER_1000_mW = 5,
-    CRSF_RF_POWER_2000_mW = 6,
+    CRSF_RF_POWER_0_mW,
```

Comment:
Assignments absolutely must be kept here, since they reference a standard. If, say, 250mw were added it would have to be at end, not in middle otherwise compatibility with protocol would be broken.